website/docs/documentation/Pipelines/Smart-seq2_Single_Nucleus_Pipeline/README.md

```diff
@@ -0,0 +1,136 @@
+| Aligner  | STAR (v.2.7.8a) | [STAR](https://github.com/alexdobin/STAR) |
+| QC Metrics | Picard (v.2.20.4) | [Broad Institute](https://broadinstitute.github.io/picard/)   |
+| Transcript Quantification | featureCounts (utilities for counting reads to genomic features) | [featureCounts(v2.0.2)](http://subread.sourceforge.net/)
+| Data Input File Format | File format in which sequencing data is provided | [FASTQ](https://academic.oup.com/nar/article/38/6/1767/3112533) |
+| Data Output File Formats | File formats in which Smart-seq2 output is provided | [BAM](http://samtools.github.io/hts-specs/), Loom (generated with [Loompy v.3.0.6)](http://loompy.org/), TXT (QC metrics and counts) |
+
+## Set-Up
+
+### Smart-seq2 Installation and Requirements
+
+The snSS2 workflow code can be downloaded by cloning the GitHub [WARP repository](https://github.com/broadinstitute/warp). For the latest release of snSS2, please see the [Smart-seq2 Single Nucleus changelog](https://github.com/broadinstitute/warp/tree/develop/pipelines/skylab/smartseq2_single_nucleus/SmartSeq2SingleNucleus.changelog.md).
+
+The workflow is deployed using [Cromwell](https://github.com/broadinstitute/cromwell), a GA4GH compliant, flexible workflow management system that supports multiple computing platforms.
+
+### Inputs
+
+There is an [example configuration (JSON) file](https://github.com/broadinstitute/warp/tree/develop/pipelines/skylab/smartseq2_single_nucleus/mouse_paired_example.json) available that you can to test the snSS2 workflow. It points to publicly available reference files and sample paired-end FASTQs.
+
+#### Sample Data Input
+
+The pipeline is designed for both single- and paired-end reads in the form of FASTQ files. The workflow accepts two FASTQ files for paired-end experiments and one FASTQ file for single-end experiments. It processes one sample (cell) at a time.
+
+*  `fastq1`: forward reads for a sample with paired-end sequencing (or reads for sample with single-end sequencing)
+*  `fastq2`: reverse reads for a sample with paired-end sequencing (not applicable for samples with single-end sequencing)
+
+The workflow uses modified tasks depending if a sample is paired-end or single-end. This is designated with a boolean, as detailed in the following Reference and Additional Inputs section.
+
+#### Reference and Additional Inputs
+
+The snSS2 workflow requires multiple references detailed in the table below. 
+* Reference inputs are created using the [BuildIndices Pipeline](https://github.com/broadinstitute/warp/tree/master/pipelines/skylab/build_indices).
+* The workflow uses a modified version of the 10x Genomics code for building the human reference package [GRCh38-2020-A](https://support.10xgenomics.com/single-cell-gene-expression/software/release-notes/build#GRCh38_2020A). 
```

Comment:
Do we also use the modified 10X code for the mouse reference?

Reply:
yes. We will update it with the new references we built.